wire-golden-files/src/main/kotlin/squareup/wire/mutable/MutableHeader.kt

```diff
@@ -0,0 +1,112 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source: squareup.wire.mutable.Header in squareup/wire/mutable_types.proto
+@file:Suppress(
+  "DEPRECATION",
+  "RUNTIME_ANNOTATION_NOT_SUPPORTED",
+)
+
+package squareup.wire.mutable
+
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.Message
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import com.squareup.wire.ReverseProtoWriter
+import com.squareup.wire.Syntax.PROTO_2
+import com.squareup.wire.WireField
+import com.squareup.wire.`internal`.JvmField
+import kotlin.Any
+import kotlin.AssertionError
+import kotlin.Boolean
+import kotlin.Deprecated
+import kotlin.DeprecationLevel
+import kotlin.Int
+import kotlin.Long
+import kotlin.Nothing
+import kotlin.String
+import kotlin.Suppress
+import kotlin.UnsupportedOperationException
+import okio.ByteString
+
+public class MutableHeader(
+  @field:WireField(
+    tag = 1,
+    adapter = "com.squareup.wire.ProtoAdapter#UINT64",
+    schemaIndex = 0,
+  )
+  public var id: Long? = null,
+  unknownFields: ByteString = ByteString.EMPTY,
```

Comment:
I think we need to make `unknownFields` be `open` on the base type so that it can be overridden here as mutable. Otherwise there's no way to clear or copy in unknown fields.

Reply:
That makes sense. Given that this requires changes to `wire-runtime`, should I make the a change in the `wire-runtime` first (a separate PR), and then land the corresponding change in `wire-kotlin-generator` ? (after having landed this).

Reply:
I think it's okay to do here. Then we correctly "blame" the change on needing to override for mutable support in case anyone goes spelunking in the future.

Reply:
Okay, made the change to `wire-runtime` and updated the generator.